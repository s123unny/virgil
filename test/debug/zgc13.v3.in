b ../core/zgc13.v3 7
b ../core/zgc13.v3 10
run
bt
info l
c
q
