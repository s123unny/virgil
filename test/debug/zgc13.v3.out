breakpoint|../core/zgc13.v3 7|0
breakpoint|../core/zgc13.v3 10|1
stopOnBreakpoint
bt|main|../core/zgc13.v3|7
variable|0|a|0|int|false
variable|1|x|#1:zgc13|zgc13|true
end
result|33
